nn-specials/src/main/java/org/frankframework/extensions/esb/EsbJmsListener.java

```diff
@@ -62,6 +63,8 @@ public class EsbJmsListener extends JmsListener implements ITransactionRequireme
 
 	private static final AppConstants APP_CONSTANTS = AppConstants.getInstance();
 	private static final String MSGLOG_KEYS = APP_CONSTANTS.getProperty("msg.log.keys");
+	static final String JMS_RR_FORCE_MESSAGE_KEY = "jms.rr.forceMessageIdAsCorrelationId.default";
```

Comment:
Since this is EsbJms specific, perhaps the propertyname prefix should reflect this? `jms.esb.*` or `esbjms.*` or whatever.

Reply:
Done!